apps/climatemappedafrica/src/lib/data/common/index.js

```diff
@@ -49,13 +67,21 @@ export async function getPageProps(api, context) {
     return null;
   }
 
+  const hurumap = await api.findGlobal("settings-hurumap");
+  const {
+    page: { value: explorePage },
+  } = hurumap;
+
   const blocks = await blockify(page.blocks, api, context);
 
   const siteSettings = await api.findGlobal("settings-site");
   const footer = getFooter(siteSettings);
+  const menus = getMenus(siteSettings);
 
   return {
     blocks,
     footer,
+    menus,
+    variant: page.slug === explorePage.slug ? "explore" : "default",
```

Comment:
I'm still not sure why we need variant on page. My understanding is:

If this page is explore page (or child of explore page):

1. Set variant on the Navigation (should show location search)
2. Set variant on the Footer (current we shouldn't display but if we're switching to always showing a footer, we won't need variant on footer)
3. Load HURUmap data and convert that into hurumap block that we can use to render the data exploration components on the FE.

What am I missing?